src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -534,6 +534,61 @@ static int server_message_init(
         return 0;
 }
 
+static int dhcp_server_append_static_hostname(
+                sd_dhcp_server *server,
+                DHCPPacket *packet,
+                size_t max_optlen,
+                size_t *offset,
+                sd_dhcp_server_lease *static_lease) {
```

Comment:
Not a big matter, but I'd like to make this take `DHCPRequest *req`, then drop `max_optlen` and `static_lease`, and find matching static lease in this function.